migrations/sqls/20250109115438-alter-document-header-add-index-down.sql

```diff
@@ -0,0 +1 @@
+/* Replace with your SQL commands */
```

Comment:
We do need a DROP index command, even though it is 99.9999999999999% certain we will never run it (and assuming there is such a thing!)

Reply:
Updated.